### PR TITLE
Adding Printing Utils

### PR DESCRIPTION
**Pull request by SNsir**

### CODE REVIEW

File: galite-core/src/main/kotlin/org/kopi/galite/util/IPPPrinter.kt [hunk 1]
```diff
@@ -15,8 +15,86 @@
...
+    TODO("Not yet implemented")
+  }
+
+  override fun setPaperFormat(paperFormat: String?) {
+    TODO("Not yet implemented")
+  }
 
+  override fun getPrinterName(): String {
+    TODO("Not yet implemented")
+  }
```
Comment (hfazai) [inline galite-core/src/main/kotlin/org/kopi/galite/util/IPPPrinter.kt:99]: This doens't have to be implemented here!

Comment (SNsir) [reply galite-core/src/main/kotlin/org/kopi/galite/util/IPPPrinter.kt:99]: ignore it , IPPPrinter non assigné

File: galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt [hunk 2]
```diff
@@ -15,8 +15,203 @@
...
+
+  /*
+   * ----------------------------------------------------------------------
+   * READ THE DONE QUEUE
+   * RETURNS A VECTOR OF FAXSTATUS
+   * ----------------------------------------------------------------------
+   */
+  fun readDoneQueue(host: String, port: Int, user: String, password: String): Vector<FaxStatus> {
+    return readQueue(host, port, user, password, "doneq")
+  }
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt:57]: change to single-expression function

File: galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt [hunk 3]
```diff
@@ -15,8 +15,203 @@
...
+
+  /*
+   * ----------------------------------------------------------------------
+   * READ THE RECEIVE QUEUE
+   * RETURNS A VECTOR OF FAXSTATUS
+   * ----------------------------------------------------------------------
+   */
+  fun readRecQueue(host: String, port: Int, user: String, password: String): Vector<FaxStatus> {
+    return readQueue(host, port, user, password, "recvq")
+  }
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt:67]: change to single-expression function

File: galite-core/src/main/kotlin/org/kopi/galite/util/ipp/IPPClient.kt [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package org.kopi.galite.util.ipp
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/ipp/IPPClient.kt:1]: can you add header

File: galite-core/src/main/kotlin/org/kopi/galite/util/ipp/IPPClient.kt [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  fun print(file: InputStream,
+            nbCopies: Int,
+            mediaAttributes: List<*>,
+            optionalAttributes: List<*>) {
+    TODO()
+  }
+  fun print(file: InputStream, nbCopies: Int, attributes: Array<String>?) {
+    TODO()
+  }
+}
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/ipp/IPPClient.kt:19]: add empty line

File: galite-core/src/main/kotlin/org/kopi/galite/util/lpr/LpR.kt [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    TODO()
+  }
+
+  /**
+   * Sets if the burst page is printed
+   */
+  open fun setPrintBurst(printBurst: Boolean) {
+    TODO()
+  }
+}
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/lpr/LpR.kt:42]: add empty line

File: galite-core/src/main/kotlin/org/kopi/galite/util/IPPPrinter.kt [hunk 7]
```diff
@@ -34,19 +34,16 @@ import org.kopi.galite.util.ipp.IPPClient
...
-                private val host: String,
-                private val port: Int,
-                private val printer: String,
-                private val user: String,
-                private val attributesForMedia: List<*>)
+                 private val host: String,
+                 private val port: Int,
+                 private val printer: String,
+                 private val user: String,
+                 private val attributesForMedia: List<*>)
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/IPPPrinter.kt:41]: `List<Array<String>>`

File: galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt [hunk 8]
```diff
@@ -147,9 +146,10 @@ object HylaFAXUtils {
    * HANDLE THE QUEUES --- ALL QUEUES ARE HANDLED BY THAT METHOD
    * ----------------------------------------------------------------------
    */
-  private fun getQueue(host: String, port: Int, user: String, password: String, qname: String): Vector<String> {
-    val entries: Vector<String>
+  private fun getQueue(host: String, port: Int, user: String, password: String, qname: String): ArrayList<String> {
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt:149]: add
 @Suppress("UNCHECKED_CAST")

File: galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt [hunk 9]
```diff
@@ -159,42 +159,44 @@ object HylaFAXUtils {
     faxClient.rcvfmt("%f| %t| %s| %p| %h| %e")
     faxClient.jobfmt("%j| %J| %o| %e| %a| %P| %D| %.25s")
     faxClient.mdmfmt("Modem %m (%n): %s")
-    entries = faxClient.getList(qname) as Vector<String>
+    entries = faxClient.getList(qname) as ArrayList<String>
```
Comment (h-haddad) [inline galite-core/src/main/kotlin/org/kopi/galite/util/HylaFAXUtils.kt:162]: ` val entries = ArrayList(faxClient.getList(qname)) as ArrayList<String>`

**Review by h-haddad — APPROVED** [review]

**Pull request merged**